command/kv_helpers.go

```diff
@@ -8,6 +8,7 @@ import (
 
 	"github.com/hashicorp/vault/api"
 	"github.com/hashicorp/vault/sdk/helper/strutil"
+	"github.com/pkg/errors"
```

Comment:
Unintended import of the pkg one?

Reply:
weird, goimports chose this one for some reason, updated.